cdci_osa_plugin/osa_lightcurve_query.py

```diff
@@ -91,7 +91,7 @@ def __init__(self,
     @staticmethod
     def ensure_timedel(du):
         # TODO: move to https://github.com/integral-observatory/ogip/
-        if 'TIMEDEL' in du.data:
+        if 'TIMEDEL' in list(du.data):
```

Comment:
Will this be always a list ? Perhaps to check if that is the case beforehand?

Reply:
it's not a list, it's `FITS_rec`, but it converts to list.
There was a deprecation recently which affects this comparison, though it is not obvious from the error.
It could be also something else.

It's true that we have not a test for this strangely and we need to add it.

Reply:
https://github.com/oda-hub/dispatcher-plugin-integral/issues/108 we'll need to check the exact result which fails. I can not do this rights.